modules/search/instant-search/components/search-results.jsx

```diff
@@ -166,15 +166,15 @@ class SearchResults extends Component {
 				aria-live="polite"
 				className="jetpack-instant-search__search-results"
 			>
-				<a
+				<button
 					className="jetpack-instant-search__overlay-close"
 					onClick={ this.closeOverlay }
 					onKeyPress={ this.onKeyPressHandler }
-					role="button"
 					tabIndex="0"
+					aria-label={ __( 'Close search results', 'jetpack' ) }
```

Comment:
Wouldn't this duplicate the aria-label already provided by the Gridicon? Or is that expected/fine?

<img width="485" alt="Screen Shot 2020-10-06 at 3 40 26 PM" src="https://user-images.githubusercontent.com/4044428/95262922-42d61480-07ea-11eb-8391-7fa3e2b672a3.png">

Reply:
Ah - it's the title on the SVG. I've added an `aria-hidden` on the SVG to address this 👍